apps/backend-e2e/src/maps.e2e-spec.ts

```diff
@@ -2240,23 +2240,6 @@ describe('Maps', () => {
         });
       });
 
-      it('should 400 if BSP file is missing', async () => {
```

Comment:
We still want to require at least one of BSP/zones/VMF. So let's invert this test to check we allow missing BSP if there's no vmf/zones, then update service logic to throw if all three are missing, and test for that in here as well.